Makefile

```diff
@@ -120,7 +121,10 @@ hypervisor:
 			echo "CONFIG_ENFORCE_VALIDATED_ACPI_INFO=y" >> $(HV_OUT)/.config;	\
 		elif [ "$(SCENARIO)" ]; then \
 			echo "CONFIG_$(shell echo $(SCENARIO) | tr a-z A-Z)=y" >> $(HV_OUT)/.config;	\
-		fi \
+		fi; \
+		if [ -f $(GENED_ACPI_INFO_HEADER) ] && [ "$(CONFIG_XML_ENABLED)" != "true" ]; then \
```

Comment:
I am fine with the patch, but such kind of warning is easily to be ignored. Can we add same statistics about the # of warning/error at the end of the build?